### ISSUE TITLE
2.12.0-RC1 branch has deprecation warning + `-Xfatal-warnings`

### ISSUE THREAD

**Issue by adriaanm** [scalamacros/paradise#86]
good: 0df362a (https://scala-ci.typesafe.com/view/scala-2.12.x/job/scala-2.12.x-integrate-community-build/672/consoleFull)

```
[macro-paradise] [warn] /home/jenkins/workspace/scala-2.12.x-integrate-community-build/target-0.9.5/project-builds/macro-paradise-c3528a3d56898fb31a2bff893401d481715be87f/tests/src/test/scala/annotations/run/issue48/Test1.scala:5: class C in package issue48 is deprecated: 
[macro-paradise] [warn]     C();
[macro-paradise] [warn]     ^
[macro-paradise] [warn] one warning found
```

bad: bce8f04 

```
[macro-paradise] [error] /home/jenkins/workspace/scala-2.12.x-integrate-community-build/target-0.9.5/project-builds/macro-paradise-cac897a86f7fb477bc566b9d8681ccf96d740fb9/tests/src/test/scala/annotations/run/issue48/Test1.scala:5: class C in package issue48 is deprecated: 
[macro-paradise] [error]     C();
[macro-paradise] [error]     ^
[macro-paradise] [error] one error found
[macro-paradise] [error] (tests/test:compileIncremental) Compilation failed
```

**Comment by adriaanm** [comment]
Not sure why `@deprecated` is needed in https://github.com/scalamacros/paradise/blob/2.12.0-RC1/tests/src/test/scala/annotations/run/issue48/Test1.scala? Worked around for RC2 community build: https://github.com/adriaanm/paradise/commit/af61abbf25482755dabcbaffd4591db554662577

**Comment by xeno-by** [comment]
I think that `@deprecated` was there just because it's an annotation. I'll replace it with some other annotation.

**Comment by xeno-by** [comment]
Fixed in paradise 2.1.0 for 2.12.0-RC2.
